OOP_HW1/src/main/java/TicketSystem.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Vector;
+
+public class TicketSystem {
+
+    public static int CheckTicketPrice(Order ticket) {
+        int price = 0; // 這裡挖空。
+        if (ticket.getQuantity() <= 0) {
+            throw new IllegalArgumentException("There is at least one ticket.");
+        }
+        switch (ticket.getTicketType()) {
+            case ADULT:
+                price = 350;
+                break;
+            case CHILD:
+                price = 280;
+                break;
+            case ELDER:
+                price = 240;
+                break;
+        }
+        return ticket.getQuantity() < 10
+                ? price * ticket.getQuantity()
+                : (int) (price * ticket.getQuantity() * 0.8);
```

Comment:
I think it have some complex logic on these statement. In this case, I will consider to wrap the logic into function or just trying to simplify the statement. 
Here's the suggestion for you:

```suggestion
        return (int) (price * ticket.GetQuantity() * (ticket.GetQuantity() >= 10 ? 0.8 : 1));
```